src/components/Pools/DetailModal/Details/index.tsx

```diff
@@ -25,10 +25,10 @@ const Details: React.FC<Props> = ({ stakingInfo }) => {
   const token1 = stakingInfo?.tokens?.[1];
   const chainId = useChainId();
   const { t } = useTranslation();
-  const totalStakedInUsd = numeral(stakingInfo?.totalStakedInUsd.toSignificant(4)).format('$0.00a');
+  const totalStakedInUsd = numeral(stakingInfo?.totalStakedInUsd?.toSignificant(4)).format('$0.00a');
```

Comment:
these values are little bit different from production values. please compare it with prod.

![Screenshot 2023-03-30 at 12 07 21 AM](https://user-images.githubusercontent.com/6604146/228635856-5d7ee66f-fe23-4c58-8454-7d48c2e855c5.png)